bin/config_helper.py

```diff
@@ -176,6 +176,14 @@ def _get_all_additional_configs(other_files):
     """
     additional_configs = dict()
     for file_path in other_files:
+        # Sanitize user input to ensure file_path is a valid Python or json file
```

Comment:
The only `config_helper` function we use in indexd is `load_json`. So I think you can remove these fence-specific functions. That's what i did in Sheepdog and Peregrine, for example https://github.com/uc-cdis/sheepdog/blob/923f796/bin/confighelper.py